src/Resources/config/admin_routing.yaml

```diff
@@ -1,4 +1,5 @@
 setono_sylius_redirect_admin_redirect:
+    prefix: /admin
```

Comment:
This prefix does not belong here. Usually it will be placed in the file that imports this routing file